src/game/stage/stage.c

```diff
@@ -411,9 +443,15 @@ void Task_GameStage(void)
         if (IS_SINGLE_PLAYER) {
             gStageFlags |= STAGE_FLAG__ACT_START;
 
+#if (GAME == GAME_SA1)
+            if (gLoadedSaveGame.timeLimitDisabled && (gGameMode == GAME_MODE_SINGLE_PLAYER || IS_MULTI_PLAYER)) {
```

Comment:
Do these match if you convert to a pointer?

Reply:
Yeah, but what would we call the macro?
Just `LOADED_SAVE` and then use it as pointer whereever possible?

Reply:
Think that would make the most sense?